cmd/pd-server/main.go

```diff
@@ -221,6 +221,8 @@ func start(cmd *cobra.Command, args []string, services ...string) {
 		exit(0)
 	}
 
+	// Check the PD version first before running.
+	server.CheckAndGetPDVersion()
```

Comment:
Is the purpose of the modification to print warning logs to help debug?

Reply:
No, it should directly panic here.

Reply:
Will panic in MustParseVersion ?

Reply:
Yes, otherwise it will panic during the leader campaign which might be too late to let the user be aware of the problem.